lemminx-liberty/src/test/java/io/openliberty/LibertyWorkspaceTest.java

```diff
@@ -17,17 +17,17 @@
 public class LibertyWorkspaceTest {
     
     @Test
-    public void readDevcMetadata() throws URISyntaxException {
+    public void testReadDevcMetadata() throws URISyntaxException {
         File srcResourcesDir = new File("src/test/resources");
         URI resourcesDir = srcResourcesDir.toURI();
         LibertyWorkspace libertyWorkspace = new LibertyWorkspace(resourcesDir.toString());
         assertNull(libertyWorkspace.getContainerName());
         assertFalse(libertyWorkspace.isContainerAlive());
-        
-        assertNotNull(libertyWorkspace.findDevcMetadata());
-        
-        assertEquals("liberty-dev", libertyWorkspace.getContainerName());
-        assertTrue(libertyWorkspace.isContainerAlive());
-    }
+        assertNull(libertyWorkspace.findDevcMetadata());    // no alive containers return null
```

Comment:
I'm confused by the test case changes. The test metadata file has a containerName `liberty-dev`, so why is line 24 checking that it is null? Is that because you changed the metadata to say the container is not alive?

Reply:
Yes, and `findDevcMetadata` returns the first alive file or null